tests/accounts.test.js

```diff
@@ -0,0 +1,760 @@
+        });
+    });
+
+    it("GET /Accounts/ID", (done) =>{
+        chai.request(server).get("/accounts/6164db5823242f430c487fca")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("object");
+            response.body.should.have.property("_id");
+            response.body.should.have.property("name");
+            response.body.should.have.property("lastname");
+            response.body.should.have.property("age");
+            response.body.should.have.property("dateBirth");
+            response.body.should.have.property("email");
+            response.body.should.have.property("username");
+            response.body.should.have.property("password");
+            response.body.should.have.property("role");
+            response.body.should.have.property("idCity");
+            done();
+        });
+    });
+
+    it("GET /Accounts/ID Not Found", (done) =>{
+        chai.request(server).get("/accounts/6164db5823242f430c487fcb")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro registro(s)");
+            done();
+        });
+    });
+
+    it("GET /Accounts/ID Bad Request", (done) =>{
+        chai.request(server).get("/accounts/6343455")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+});
+
+describe("POST Tests Accounts",()=>{
+    let idAccount = null;
+
+    it("POST /Accounts Bad Request All", (done) =>{
+        const account = {};
+        chai.request(server).post("/accounts").send(account)
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+    
+    it("POST /Accounts Bad Request Name", (done) =>{
+        const account = {
+            lastname: 354,
+	        name: 123,
+	        age: 21,
+	        dateBirth: "2000-08-05",
+	        email: "miroslava_19_7@outlook.es",
+	        username: "Miroslava123",
+	        password: "Mmol180515#",
+	        role: "user",
+	        idCity: "61645a7c0879b0c15c7116f7"
+        };
+        chai.request(server).post("/accounts").send(account)
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("POST /Accounts Bad Request Age", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("POST /Accounts Bad Request Age", (done) => {
```